packages/roosterjs-content-model-dom/lib/domToModel/processors/childProcessor.ts

```diff
@@ -37,14 +37,19 @@ export const childProcessor: ElementProcessor<ParentNode> = (
  * @param group The parent block group
  * @param parent Parent DOM node to process
  * @param context DOM to Content Model context
+ * @param processNonVisibleElements If true elements that has display:none style will be processed @default false
  *
  */
 export function processChildNode(
     group: ContentModelBlockGroup,
     child: Node,
-    context: DomToModelContext
+    context: DomToModelContext,
+    processNonVisibleElements: boolean = false
 ) {
-    if (isNodeOfType(child, 'ELEMENT_NODE') && child.style.display != 'none') {
+    if (
+        isNodeOfType(child, 'ELEMENT_NODE') &&
+        (child.style.display != 'none' || processNonVisibleElements)
+    ) {
         context.elementProcessors.element(group, child, context);
```

Comment:
do we save the display value today? if not, it will be rendered without "display:none" so user can see it

Reply:
We save 
<img width="254" height="313" alt="image" src="https://github.com/user-attachments/assets/5e04e716-20f8-4481-a143-c203aa4e0d95" />